### DISCUSSION TITLE
flutter app

### DISCUSSION THREAD

**Discussion by MadHunter58** [community/community#146736] [Programming Help]
### Select Topic Area

Question

### Body

hey i wanted to make music app using flutter and supabase and i got problem the song never play in my project and i made the mp3 and everything when i select the song name will display from database and and the song not ,i have bucket have the songs and made it public and i copy the public url and put it in my table and still the song not run anyone can hlep me :(

**Comment by taki-on** [comment]
@MadHunter58 Hello! this discussion board is designed for giving feedback about GitHub services. If you have questions about Flutter, you can check out a Flutter community on the Internet to ask your questions :)

**Comment by jonhubby** [comment]
Make sure you've set up the proper audio permissions and dependencies in your app. You'll need to add the just_audio package and handle the necessary permissions. Double-check the URLs you're using to access the songs in your Supabase bucket. Ensure the URLs are correct and that the songs are publicly accessible.

**Comment by ebndev** [comment]
Hi @MadHunter58, thanks for posting in GitHub Discussions! 

I've gone ahead and moved this post to our `Programming Help` category, as this topic may be more relevant there and might help someone give you a nudge in the right direction. 😁
